boa/vm/py_evm.py

```diff
@@ -565,13 +565,13 @@ def get_storage_slot(self, address: Address, slot: int) -> bytes:
         return data.to_bytes(32, "big")
 
 
-GENESIS_PARAMS = {"difficulty": constants.GENESIS_DIFFICULTY, "gas_limit": int(1e8)}
+GENESIS_PARAMS = {"gas_limit": int(1e8)}
 
 
 # TODO make fork configurable - ex. "latest", "frontier", "berlin"
 # TODO make genesis params+state configurable
 def _make_chain():
     # TODO should we use MiningChain? is there a perf difference?
     # TODO debug why `fork_at()` cannot accept 0 as block num
-    _Chain = chain.build(MainnetChain, chain.latest_mainnet_at(1))
+    _Chain = chain.build(MiningChain, chain.latest_mainnet_at(0))
```

Comment:
I would definitely recommend using `MiningChain` over `MainnetChain` for stability and your won control. Alternatively, you can configure the chain directly:

```py
_Chain = MiningChain.configure(chain_id=1, vm_configuration=[(0, PragueVM)])
```

Seeing as one of the comments above has a goal to make this configurable, you could eventually configure it as, for example, Cancun to Prague at 5:

```py
vm_configuration=[(0, CancunVM), (5, PragueVM)])
```